gammapy/detect/lima.py

```diff
@@ -2,13 +2,68 @@
 import copy
 import logging
 import numpy as np
+from astropy.coordinates import Angle
+from astropy.convolution import Tophat2DKernel
 from gammapy.stats import significance, significance_on_off
+from gammapy.cube import MapDataset, MapDatasetOnOff
 
-__all__ = ["compute_lima_image", "compute_lima_on_off_image"]
+__all__ = [
+    "SignificanceMapEstimator",
+    "compute_lima_image",
+    "compute_lima_on_off_image",
+]
 
 log = logging.getLogger(__name__)
 
 
+class SignificanceMapEstimator:
```

Comment:
I have a slight preference for naming this `LiMaMapEstimator` or similar, because it refers to a very specific (even if it's standard...) method of estimating significance. In general there are other methods. In many ways I see this `MapEstimator` on the same level as a `TSMapEstimator` (in fact I think one can show that the Li&Ma solution corresponds to the TS map case of constant exposure, background and flat source kernel...).

In this regard I think it's also an option to extend the number of  maps computed by the `LiMaMapEstimator` to more quantities, such as `flux`, `flux_err` and `flux_ul` later.

Reply:
Done. 
But I guess in the long run this might not be the most relevant name for this type of `Estimator`.